src/getConfig.js

```diff
@@ -1,8 +1,14 @@
 /* eslint no-console: 0 */
 /* eslint no-prototype-builtins: 0 */
+const chalk = require('chalk')
+const format = require('stringify-object')
 const intersection = require('lodash/intersection')
 const defaultsDeep = require('lodash/defaultsDeep')
 const isObject = require('lodash/isObject')
+const validate = require('jest-validate').validate
+const unknownOptionWarning = require('jest-validate/build/warnings').unknownOptionWarning
+const logValidationWarning = require('jest-validate/build/utils').logValidationWarning
```

Comment:
You can change this to `const logValidationWarning = require('jest-validate').logValidationWarning`